app/src/main/java/eu/kanade/tachiyomi/ui/manga/track/TrackingBottomSheet.kt

```diff
@@ -128,20 +138,31 @@ class TrackingBottomSheet(private val controller: MangaDetailsController) : Bott
     override fun onStatusClick(position: Int) {
         val item = adapter?.getItem(position) ?: return
         if (item.track == null) return
+        if (controller.isNotOnline()) {
+            sheetBehavior.hide()
+            return
+        }
 
         SetTrackStatusDialog(this, item).showDialog(controller.router)
     }
 
     override fun onChaptersClick(position: Int) {
         val item = adapter?.getItem(position) ?: return
         if (item.track == null) return
-
+        if (controller.isNotOnline()) {
+            sheetBehavior.hide()
+            return
+        }
         SetTrackChaptersDialog(this, item).showDialog(controller.router)
     }
 
     override fun onScoreClick(position: Int) {
         val item = adapter?.getItem(position) ?: return
         if (item.track == null) return
+        if (controller.isNotOnline()) {
+            sheetBehavior.hide()
```

Comment:
these ` sheetBehavior.hide()` should probably be `dismiss()` instead, not sure if calling hide also dismisses BottomSheetDialog